app/validators/form_password_validator.rb

```diff
@@ -2,8 +2,20 @@ module FormPasswordValidator
   extend ActiveSupport::Concern
 
   included do
+    attr_accessor :password
+    attr_reader :user
+
     validates :password,
               presence: true,
               length: Devise.password_length
+
+    validate :valid_password
+  end
+
+  def valid_password
+    return if errors[:password].any?
+    temp_user = User.new(password: password)
+    return if temp_user.valid?
+    errors.add :password, temp_user.errors[:password]
```

Comment:
I'm reluctant to remove the explicit minimum length requirement validation. The zxcvbn library is great, but it becomes a single point of failure for something so central to our security. If for some reason the library changed its internal definitions, or we changed the config setting, we would lose what is a basic mathematical requirement (min length).

We might also in future want to bump up the minimum (e.g. `10`) and exceed what zxcvbn is doing. The belt-and-suspenders approach seems legit here to me.